src/w3c/templates/headers.js

```diff
@@ -190,7 +190,7 @@ export default (conf, options) => {
    */
   sub("beforesave", doc => {
     const details = doc.querySelector(".head details");
-    details.open = true;
+    if (details) details.open = true;
```

Comment:
Would this work too? 
```suggestion
    details?.open = true;
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Invalid_assignment_left-hand_side#using_optional_chaining_as_assignment_target

https://github.com/tc39/proposal-optional-chaining-assignment

Reply:
Right, I had my suspicions